layout_padded/layout_padded.md

```diff
@@ -1934,21 +2074,35 @@ is representable as a value of type `size_t`,
 and is representable as a value of type `index_type`.
 
 ```c++
-static constexpr size_t @_static-padding-stride_@ = /* see-below */; // exposition only
+static constexpr size_t @_static-padding-stride_@ = /* @_see-below_@ */; // @_exposition only_@
 ```
 
 [4]{.pnum} The value is
 
-  * [4.1]{.pnum } `0`, if _`rank_`_
+  * [4.1]{.pnum} `0`, if `extents_type::rank()`
       equals zero or one; otherwise
 
-  * [4.2]{.pnum } the `size_t` value which is
+  * [4.2]{.pnum} `dynamic_extent`, if `padding_value` or _`last-static-extent`_ equals `dynamic_extent`; otherwise
+
+  * [4.3]{.pnum } the `size_t` value which is
     the least multiple of `padding_value`
     that is greater than or equal to
-    _`last-static-extent`_, if
-      `padding_value` does not equal `dynamic_extent` and _`last-static-extent`_ does not equal `dynamic_extent`; otherwise
+    _`last-static-extent`_.
 
-  * [4.3]{.pnum } `dynamic_extent`.
+<!--
+NOTE (mfh 2024/01/18) Above comments on
+`layout_left_padded::mapping::`_`stride-1`_
+also apply to _`stride-rm2`_.
+-->
+
+```c++
+index_type /* @_see below_@ */ @_stride-rm2_@ = @_static-padding-stride_@; // @_exposition only_@
+```
+
+[5]{.pnum} *Recommended practice*: Implementations should not store
+this value if _`static-padding-stride`_ is not `dynamic_extent`.
+For example, using `extents<index_type,`_`static-padding-stride`_`>`
+instead of `index_type` as the type of _`stride-rm2`_ would achieve this.
```

Comment:
LWG mailing list suggestion (please change manually, not through github):

```suggestion
Using `extents<index_type,`_`static-padding-stride`_`>`
instead of `index_type` as the type of _`stride-rm2`_ would achieve this.
```